pkg/sdn/api/types.go

```diff
@@ -44,7 +44,7 @@ type NetNamespace struct {
 	kapi.ObjectMeta
 
 	NetName string
-	NetID   uint
+	NetID   *uint
```

Comment:
Talked on IRC. I think the outcome is to use `*uint, omitempty` for the external type

* Old oadm clients didn't have a way to request unsetting a net namespace
* An empty netid is quickly filled by allocation, so is not empty long-term
* If upgrade of nodes and masters is completed prior to use of `oadm pod-network isolate-projects`, no missing values will be encountered by old masters/nodes
* If a missing netid is encountered by an old master/node, it will be interpreted as netid 0 and be treated as a global project until a netid is allocated and set, at which point it will be isolated.